spec/requests/groups_controller_spec.rb

```diff
@@ -0,0 +1,108 @@
+require 'rails_helper'
+
+RSpec.describe 'Groups', type: :request do
```

Comment:
Yes, I'm planning to add a set of Capybara tests to cover all the new features in the Group Management System.